File: report-service/src/main/kotlin/ru/morkovka/report/entity/ReleaseNote.java
package ru.morkovka.report.entity;

import java.util.ArrayList;

public class ReleaseNote {
    private String distributions;
    private ArrayList<String> changes;
    private ArrayList<String> dbChanges;
    private ArrayList<String> configs;
    private ArrayList<String> installation;
    private ArrayList<String> testing;
    private ArrayList<String> rollback;

    /**
     * Дистрибутивы
     * **/
    public String getDistributions() {
        return distributions;
    }

    public void setDistributions(String distributions) {
        this.distributions = distributions;
    }

    /**
     * Новые функции и исправления
     * **/
    public ArrayList<String> getChanges() {
        return changes;
    }

    public void setChanges(ArrayList<String> changes) {
        this.changes = changes;
    }

    /**
     * Настройки и изменения в структуре БД
     * **/
    public ArrayList<String> getDbChanges() {
        return dbChanges;
    }

    public void setDbChanges(ArrayList<String> dbChanges) {
        this.dbChanges = dbChanges;
    }

    /**
     * Конфигурация
     * **/
    public ArrayList<String> getConfigs() {
        return configs;
    }

    public void setConfigs(ArrayList<String> configs) {
        this.configs = configs;
    }

    /**
     * Порядок установки и изменения настроек
     * **/
    public ArrayList<String> getInstallation() {
        return installation;
    }

    public void setInstallation(ArrayList<String> installation) {
        this.installation = installation;
    }

    /**
     * План тестирования
     * **/
    public ArrayList<String> getTesting() {
        return testing;
    }

    public void setTesting(ArrayList<String> testing) {
        this.testing = testing;
    }

    /**
     * План отката
     * **/
    public ArrayList<String> getRollback() {
        return rollback;
    }

    public void setRollback(ArrayList<String> rollback) {
        this.rollback = rollback;
    }
}

File: report-service/src/main/kotlin/ru/morkovka/report/entity/ReleaseNote.kt
package ru.morkovka.report.entity

data class ReleaseNote(
    /**
     * Задача в нашей джире
     */
    var taskInKey: String,

    /**
     * Исходные коды
     */
    var sourceCode: MutableMap<String, MutableList<String>>,

    /**
     * Артефакты
     */
    var artifact: MutableMap<String, MutableList<String>>,

    /**
     * Новые функции и исправления
     */
    var features: MutableMap<String, TaskFeature>,

    /**
     * Настройки и изменения в структуре БД
     */
    var dbChanges: MutableMap<String, MutableList<String>>,

    /**
     * Информация для подключений и мониторинга
     */
    var monitoringChanges: MutableMap<String, MutableList<String>>,

    /**
     * Конфигурация
     */
    var configs: MutableMap<String, MutableList<String>>,

    /**
     * Порядок установки и изменения настроек
     */
    var deploy: MutableMap<String, MutableList<String>>,

    /**
     * План тестирования
     */
    var testCase: MutableMap<String, MutableList<String>>,

    /**
     * План отката
     */
    var rollback: MutableMap<String, MutableList<String>>
)